src/app/helpers/announcement/controller.js

```diff
@@ -17,14 +16,11 @@ let ApiInterceptor = require('sb_api_interceptor')
 
 class AnnouncementController {
 
-  constructor() {
-    //table name should be same as the name in database table
-    let tableMapping = {
-      'announcement': AnnouncementModel,
-      'announcementtype': AnnouncementTypeModel,
-      'attachment': AttachmentModel,
-      'metrics': MetricsModel
-    }
+  constructor({metrics, announcement, announcementtype } = {}) {
```

Comment:
this constructor needs documentation to explain the object passed in and the properties which they must have.